### PR TITLE
Django-environ configuration

### LINKED ISSUE #24: Configure django-environ
### The math
Read about usage of environment variables in PaaS and heroku/travis for deployment.

### The task
* [x] Implement single `settings.py` using [django-environ](https://django-environ.readthedocs.io/en/latest/)
  * [x] Remove everything related to `local_settings.py`
* [x] Make every option differing at development and production be read from environment variables
  * [x] DEBUG
  * [x] SECRET
  * [x] db config
  * [x] use `env.Path` for constructing fs pathes
  * [x] etc.
* [x] Enable reading env vars from dotenv (`.env`) file
  * [x] .gitignore it
  * [x] add `.env.example` file listing required env vars with dummy values for the other dev as a quickstart

### PR DESCRIPTION
**Pull request by pavlenk0**
Close #24

### CODE REVIEW

**Review by Miseracle — APPROVED** [review]
Will you use pep-257 hook later?

**Comment by pavlenk0** [comment]
@Miseracle Yes. I will add this later. I updated my hooks to the latest version automatically by running `pre-commit autoupdate` and it rewrote .pre-commit-config.yaml file without comments. Did you get the message in your terminal about changes in [github.com/Lucas-C/pre-commit-hooks](https://github.com/Lucas-C/pre-commit-hooks)?

**Review by klochenok — APPROVED** [review]

File: mysite/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DEBUG=on
+DJANGO_SETTINGS_MODULE=mysite.settings
+SECRET_KEY=key
```
Comment (MaxymVlasov) [inline mysite/.env.example]: Need new line on the end of file

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,38 +1,33 @@
-- repo: git://github.com/pre-commit/pre-commit-hooks
```
Comment (MaxymVlasov) [inline .pre-commit-config.yaml:1]: WTF with diff?

**Review by MaxymVlasov — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 2 above)
Comment (pavlenk0) [reply .pre-commit-config.yaml:1]: See my comment above to get the answer for your question.

File: mysite/.env.example (hunk 1 above)
Comment (pavlenk0) [reply mysite/.env.example]: Thank you. I will fix this in the next time.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (webknjaz) [reply .pre-commit-config.yaml:1]: `.gitattributes`

Comment (pavlenk0) [reply .pre-commit-config.yaml:1]: @webknjaz Sorry, I don't understand what do you mean.

**Review by webknjaz — CHANGES_REQUESTED** [review]
fix pre-commit's config and delete local_settings from gitattributes

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 dj-database-url==0.4.1
```
Comment (webknjaz) [inline requirements.txt]: This one is already a dependency of `django-environ`

Comment (pavlenk0) [reply requirements.txt]: Ok. Thanks

**Review by DrEdi — APPROVED** [review]

File: blog/templates/blog/base.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
             {% else %}
                 <a href="{% url 'login' %}" class="top-menu"><span class="glyphicon glyphicon-lock"></span></a>
             {% endif %}
-            <h1><a href="/">Django Girls Blog</a></h1>
+            <h1><a href="/">Django Girls blog</a></h1>
```
Comment (webknjaz) [inline blog/templates/blog/base.html]: This change is unrelated.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by webknjaz — APPROVED** [review]

**Pull request merged**